docs/streams/stream-tasks/view-streams.md

```diff
@@ -20,7 +20,115 @@ View existing streams.
 
 1. To see more details and stream statistics, click a stream name.
 
-When you view a stream in the console, you will not see a backlog of messages. This screen does not persist, which means that when it is open, you will see new messages as they are sent. If you close the screen, any messages displayed will not be visible the next time that you open it.
+</TabItem>
+<TabItem value="py" label="Python SDK">
+
+View a list of existing streams.
+
+```python
+from c8 import C8Client
+
+# Define variables
+URL = "play.paas.macrometa.io"
+GEO_FABRIC = "_system" # Change this to your fabric
+API_KEY = "your-api-key" # Change this to your API key
+
+# Create client instance with API KEY
+client = C8Client(protocol='https', host=URL, port=443, apikey=API_KEY, geofabric=GEO_FABRIC)
+
+# Fetch the list of streams
+streams = client.get_streams()
+
+# Print the formatted list of streams
+print("List of streams:")
+for stream in streams:
+    print(f"- Name: {stream['name']}")
+    print(f"  Topic: {stream['topic']}")
+    print(f"  Type: {stream['type']}")
+    print(f"  Status: {stream['status']}")
+```
+
+Get stats for a specific stream.
+
+```python
+from c8 import C8Client
+
+# Define variables
+URL = "play.paas.macrometa.io"
+GEO_FABRIC = "_system" # Change this to your fabric
+API_KEY = "your-api-key" # Change this to your API key
+
+# Create client instance with API KEY
+client = C8Client(protocol='https', host=URL, port=443, apikey=API_KEY, geofabric=GEO_FABRIC)
+
+# Name of the stream to get information for
+# Do not include the prefix
+stream_name = "your-stream-name"
+
+try:
+    # Get stream stats
+    stream_stats = client.get_stream_stats(stream_name)
+
+    # Print stream stats
+    print("Stream statistics for '{}'".format(stream_name))
+    print("----------------------------------------")
+    for key, value in stream_stats.items():
+        print("{}: {}".format(key, value))
+        
+except c8.exceptions.StreamNotFoundError:
+    print("Stream '{}' not found".format(stream_name))
```

Comment:
```suggestion
    print("Error getting stream stats: {}".format(e))
```

Reply:
With the above two suggestions it will work fine and the issue which @Luba89 faced in the comment just above it will be resovled